Competition2018/src/main/java/competition/subsystems/elevator/commands/SetElevatorTargetHeightCommand.java

```diff
@@ -0,0 +1,40 @@
+package competition.subsystems.elevator.commands;
+
+import xbot.common.command.BaseSetpointCommand;
+import xbot.common.command.SupportsSetpointLock;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.elevator.ElevatorSubsystem;
+
```

Comment:
remove blank line

Reply:
There should only be 1 blank line between the end of the imports and the start of the class declaration

Reply:
done